src/routes/(dashboard)/PublisherDashboard.svelte

```diff
@@ -61,8 +67,10 @@
 
     let selectedReviewContentIds: number[] = [];
     let selectedInProgressContentIds: number[] = [];
+    let selectedCommunityReviewContentIds: number[] = [];
```

Comment:
I don't think this extra array here is necessary. The reason for having the two arrays is so that we can split them and send two requests, one to each endpoint. Since these community review content ids will be sent to the same endpoint as the regular review content ids, I think we can get away with using that one.

In fact, I think we can make both arrays local variables in the function below. They used to be used globally but now are only used in the `assignContent` function so we could reduce the number of global vars up here.

Reply:
Yeah, this felt gross extending the fn in this manner but it was the most straightforward way. I figured I'd put it out there and get some feedback. I'll refactor to remove the duplication and bring those variables into the function scope.